ios/start.md

```diff
@@ -10,7 +10,7 @@ Build an iOS app using the AWS Amplify CLI and the AWS SDK for iOS. The Amplify
 
 [Install and configure the Amplify CLI](..)
 
-[Install Xcode](https://developer.apple.com/xcode/downloads/) version 9.2 or later.
+[Install Xcode](https://developer.apple.com/xcode/downloads/) version 10.2 or later.
```

Comment:
Why is an Xcode upgrade needed?

Reply:
i saw from here https://github.com/awslabs/aws-mobile-appsync-sdk-ios
Note: AWS AppSync uses Swift 4.2.1. Use Xcode 10.2 or greater to build.

Reply:
Hmm that might be wrong. @royjit what do you think?

Reply:
Looks like AWS SDK will work for Xcode >= 9.2 except for AppSync GraphQL which require >=10.2.

Reply:
Is there a reason for this? It basically means any customer would need to default to the highest water mark when using multiple services.

Reply:
I recall seeing one of the dependencies that AppSync uses required the later Swift version. For developers with Mac's on 10.12.6 (Sierra), they can only install up to XCode 9.2.1, so if they go to step 4 and add Appsync, it will fail. they will have to upgrade their OS and xcode to get step 4 working. I think we should just convey the information in step 4 rather than have them debug the problem, which is that you will need xcode 10.2 for AppSync. If they don't need to use appsync, they can move onto Next Steps (Auth, storage, etc..). what do you guys think?

Reply:
Another way to reframe the discussion is how consistent we want to be in our docs. We can either document the minimum required versions necessary for all supported SDKs to run (in this case, Xcode 10.2.1), or we can document the minimum for *most* of the SDKs, and add exceptions where needed. I'd favor consistency, and the newer version, since it reduces complexity and cognitive burden, but I'm unclear how many legacy customers that would impact, so I could be talked out of that.

Reply:
thanks for all chiming in, I think we can close on this with a brief meeting from tim's last point. i'll set this up